tests/dummy/app/view-components/nrg-lightbox/template.hbs

```diff
@@ -2,15 +2,15 @@
 
 <NrgCodeBlock @label="Lightbox Thumbnails" @codeBlockName="lightbox-thumbnails.hbs">
   <!-- BEGIN-SNIPPET lightbox-thumbnails -->
-  {{#each model as |photo|}}
+  {{#each photos as |photo|}}
     <NrgLightboxThumbnail @photo={{photo}} />
   {{/each}}
   <!-- END-SNIPPET  -->
 </NrgCodeBlock>
 
 <NrgCodeBlock @label="Lightbox Thumbnails with Custom Details" @codeBlockName="lightbox-thumbnails-block.hbs">
   <!-- BEGIN-SNIPPET lightbox-thumbnails-block -->
-  {{#each model as |photo|}}
+  {{#each photos as |photo|}}
     <NrgLightboxThumbnail @photo={{photo}}>
       <strong>{{photo.detail}}</strong>
       <span>{{counter}}</span>
```

Comment:
```suggestion
      <span>{{@counter}}</span>
```

Reply:
This is in a route, and does not need the `@` syntax.

Reply:
Do you want me to make those changes here, or do you want to make the route changes (`@/this.`) in you PR?

Reply:
I'll take them.